src/viam/media/viam_rgba_plugin.py

```diff
@@ -55,7 +55,7 @@ def _open(self):
         width = int.from_bytes(header[4:8], "big")
         height = int.from_bytes(header[8:12], "big")
         self._size = width, height
-        self.mode = "RGBA"
+        self._mode = "RGBA"
```

Comment:
I believe that's for existing images, but not for creating our own image decoder. The [documentation](https://pillow.readthedocs.io/en/stable/handbook/writing-your-own-image-plugin.html) states
>An image plugin should contain a format handler derived from the [PIL.ImageFile.ImageFile](https://pillow.readthedocs.io/en/stable/reference/ImageFile.html#PIL.ImageFile.ImageFile) base class. This class should provide an _open method, which reads the file header and set at least the internal _size and _mode attributes so that [mode](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.mode) and [size](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.size) are populated.

so we should be safe setting these private attrs here. (famous last words, given this PR)